docs/glossary.md

```diff
@@ -0,0 +1,79 @@
+---
+# Glossary position should be as last as possible, but before the Resources category.
+sidebar_position: 999
+---
+
+# Glossary
+
+Use the glossary to find definitions and further information for new or
+ambiguous terms.
+
+### Aggregator
+
+A server that implements the Distributed Aggregation Protocol (DAP). Divvi Up
+and its partners operate DAP-compliant aggregators. Aggregators can be
+self-hosted.
+
+Aggregators ingest measurements sent from clients. They perform aggregation of
+batches or reports into aggregations, and service [collector](#collector)
+requests.
+
+Aggregators can be in one of two roles, [Leader](#leader) or [Helper](#helper).
+
+Examples of aggregator software include [Janus (ISRG)][janus], and [Daphne
+(Cloudflare, helper-only)][daphne].
+
+### Client
+
+An end-user application whose developer wishes to collect metrics on.
```

Comment:
I took a crack at rewriting this, but I don't know if I'm 100% happy with it.
```suggestion
Software that uses the DAP protocol to contribute a measurement to a metric. Typically, application developers will include a DAP client library in end-user software or a webapp.
```

Reply:
I landed on
> An end-user app that uses DAP to contribute a measurement to a metric. Typically, app developers can achieve this using a DAP client library.